packages/telnyx_webrtc/lib/call.dart

```diff
@@ -82,10 +82,7 @@ class Call {
         sessid: _sessid);
 
     var byeMessage = SendByeMessage(
-        id: uuid.toString(),
-        jsonrpc: "2.0",
-        method: SocketMethod.BYE,
-        params: byeParams);
+        id: uuid, jsonrpc: "2.0", method: SocketMethod.BYE, params: byeParams);
```

Comment:
Just a suggestion:
`jsonrpc: 2.0` is the version right?
I can see it is being used in multiple places. We can define it somewhere in the constants so that there is no hassle if we need to change this value.

Reply:
Good idea, I will add a default value in the classes